### PR TITLE
ADD chihuahuawifhat/baddog.png to chihuahua chain registry

### PR DESCRIPTION
**Pull request by SmolFatality**

### CODE REVIEW

File: chihuahua/assetlist.json [hunk 1]
```diff
@@ -57,5 +57,31 @@
...
+          "exponent": 0
+        },
+        {
+          "denom": "achihuahuawifhat",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/chihuahua1x4q2vkrz4dfgd9hcw0p5m2f2nuv2uqmt9xr8k2/achihuahuawifhat",
+      "name": "Chihuahuawifhat",
+      "display": "chihuahuawifhat",
```
Comment (JeremyParish69) [inline chihuahua/assetlist.json:76]: there is no unit of this name defined. Is this supposed to be 'achihuahuawifhat'?

File: chihuahua/assetlist.json [hunk 2]
```diff
@@ -57,5 +57,31 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/puppyhuahua_logo.png"
       }
     }
-  ]
-}
\ No newline at end of file
+  ],
+
```
Comment (JeremyParish69) [inline chihuahua/assetlist.json:61]: I think you messed up the formatting a bit. Can you please restore this to the correct schema?

File: chihuahua/assetlist.json (hunk 1 above)
Comment (SmolFatality) [reply chihuahua/assetlist.json:76]: Yes sorry,

File: chihuahua/assetlist.json (hunk 2 above)
Comment (SmolFatality) [reply chihuahua/assetlist.json:61]: Yes

File: chihuahua/assetlist.json (hunk 1 above)
Comment (JeremyParish69) [reply chihuahua/assetlist.json:76]: I think the name was better before. And the `display` value is still invalid.

File: chihuahua/assetlist.json (hunk 2 above)
Comment (JeremyParish69) [reply chihuahua/assetlist.json]: I think the issue was that you added another asset object but outside the `assets` array. This is still not following the schema correctly

Comment (SmolFatality) [reply chihuahua/assetlist.json]: okay, i see what you mean, i'll correct it need to edit it.

Comment (SmolFatality) [reply chihuahua/assetlist.json]: corrected. hope this is the last review you will have to do. sorry in advance!

**Pull request merged**